src/server/components/dashboard/listsItems/item/update/actionHandlers/helpers.ts

```diff
@@ -17,3 +18,16 @@ export async function sendPublishedEmail(listItem: ListItemWithAddressCountry) {
     searchLink
   );
 }
+
+export async function sendUnpublishEmail(list: List) {
+  if (list?.jsonData?.users) {
```

Comment:
could you add a log incase list?.jsonData.users is empty please? it might be the case that res.locals is missing list data, or changed along the way, so worth logging. 


I think ideally we just pass in listId to this function and get it to get the list of users itself rather than having to pass down a list item, but I think it's fine as is

Reply:
I guess my thought was the less queries the better but it would be good to know why you think it's better to have the listId passed in as an argument?